backend/app.py

```
@@ -1673,6 +1673,119 @@ async def delete_team_mapping(
+            "task_id": task.id,
+            "status_url": f"/api/matches/task/{task.id}",
+            "match": {
+                "home_team": match_data.home_team,
+                "away_team": match_data.away_team,
+                "match_date": match_data.match_date
+            }
+        }
+
+    except Exception as e:
+        logger.error(f"Error submitting match for async processing: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=f"Failed to queue match: {str(e)}")
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)